src/script/campaign/mod/vco-listeners.lua

```diff
@@ -2,6 +2,29 @@ local vco = core:get_static_object("vco");
 
 -- COMMON --
 
+-- POOLED RESOURCES --
```

Comment:
`-- COMMON --` is already meant to be category, I do not think we are going to have that much of each type to have different sub categories. I am all up to create a library for common stuff in the future tho.

Reply:
I've already mentioned that this file will get really long and ugly in future, so I think we'll want _some_ form of library. For now I'd be fine with additional comment headings as in this branch.